app/geo/gui/src/ts/twxplore/gui/geo/main.tsx

```diff
@@ -21,14 +24,17 @@ const browserHistory = createBrowserHistory();
 ReactDOM.render(
     <ApolloProvider client={apolloClient}>
       <ApolloHooksProvider client={apolloClient}>
-        <LoggerContext.Provider value={logger}>
-          <Router history={browserHistory}>
-              <Switch>
-                  <Route exact path={Hrefs.home} component={Home}/>
-                  <Route component={NoRoute}/>
-              </Switch>
-          </Router>
-        </LoggerContext.Provider>
+        <Provider store={store}>
+          <LoggerContext.Provider value={logger}>
+            <Router history={browserHistory}>
+                <Switch>
```

Comment:
Some people prefer to break these out into a separate component <Routes>. Up to you.